src/providers/ec2.ts

```diff
@@ -60,11 +60,51 @@ EOF
   /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -s -c file:/tmp/log.conf || exit 2
 }
 action () {
-  if [ "$(< RUNNER_VERSION)" = "latest" ]; then RUNNER_FLAGS=""; else RUNNER_FLAGS="--disableupdate"; fi
-  sudo -Hu runner /home/runner/config.sh --unattended --url "https://{}/{}/{}" --token "{}" --ephemeral --work _work --labels "{},cdkghr:started:\`date +%s\`" $RUNNER_FLAGS --name "{}" || exit 1
+  # Define variables for injected parameters
+  taskToken="$1"
```

Comment:
Are you sure about this? `$1` looks like a bash variable, but `{}` comes from `stepfunctions.JsonPath.format()` below that expects `{}` in the format string.

Reply:
No I'm not sure but would love to bring it to a better readable format.
This is hard to read...